tools/shoestring/Jenkinsfile

```diff
@@ -1,9 +1,10 @@
 defaultCiPipeline {
-	operatingSystem = ['ubuntu']
+	// First OS is the default for CI
+	operatingSystem = ['ubuntu', 'fedora']
```

Comment:
should we add windows?

Reply:
Yea, I started to look at Windows a while back and tests were crashing Python 3.11. 😅 
I was planning to revisit the branch now that we moved to Python 3.12
https://github.com/symbol/product/tree/fix/init_on_windows